rotkehlchen/db/schema.py

```diff
@@ -643,6 +643,13 @@
 );
 """
 
+DB_CREATE_CONNECTED_LOCATIONS = """
+CREATE TABLE IF NOT EXISTS connected_location (
```

Comment:
Hmm why do this?  There is no need for a new table. This approach is wrong. You already have all data inside the DB. Query their location with a `DISTINCT` attribute and you are done.